src/Combinatorics/Graphs/functions.jl

```diff
@@ -478,10 +722,30 @@ function has_edge(g::Graph{T}, e::Edge) where {T <: Union{Directed, Undirected}}
     return has_edge(g, e.source, e.target)
 end
 
+@doc raw"""
+    has_directed_edge(g::MixedGraph, source::Int64, target::Int64)
+
+Check for an edge in a graph.
+
+# Examples
```

Comment:
there is an example heading without an example afterwards. also in a few other places in this PR